setup.py

```diff
@@ -1,14 +1,12 @@
-from setuptools import find_packages, setup
-
-with open("README.rst") as f:
-    long_description = f.read()
+from pathlib import Path
 
+from setuptools import find_packages, setup
 
 setup(
     name="web-poet",
     version="0.18.0",
     description="Zyte's Page Object pattern for web scraping",
-    long_description=long_description,
+    long_description=Path("README.rst").read_text(),
```

Comment:
Not an issue specific to these changes, but this may be a good opportunity to specify the encoding, for cross-platform support (not sure when that would become an issue, but it has hit us in the past precisely in setup.py).